miniapp/src/main/AndroidManifest.xml

```diff
@@ -22,13 +22,13 @@
             tools:replace="android:initOrder" />
 
         <provider
-            android:name="com.rakuten.tech.mobile.miniapp.display.MiniAppDownloadedFileProvider"
-            android:authorities="${applicationId}.miniapp.downloadedfileprovider"
+            android:name="com.rakuten.tech.mobile.miniapp.display.MiniAppDefaultFileProvider"
```

Comment:
Could you also add a rule to the `proguard-rules.pro` for this class? I should have done that earlier when I added this 😅 

I think something like this?
```
-keep class com.rakuten.tech.mobile.miniapp.display.MiniAppDefaultFileProvider { *; }
```